src/CardComponent.js

```diff
@@ -0,0 +1,16 @@
+/** display user information in a card */
+const CardComponent = ({members}) =>{
```

Comment:
You can destructure here only like

{members:{img,name,city,company}}

Reply:
This suggestion is really good.